src/pages-and-resources/discussions/app-config-form/apps/shared/discussion-topics/TopicItem.jsx

```diff
@@ -17,6 +17,8 @@ const TopicItem = ({
 }) => {
   const [title, setTitle] = useState(name);
   const [showDeletePopup, setShowDeletePopup] = useState(false);
+  const [collapseIsOpen, setCollapseIsOpen] = useState(!name.length);
```

Comment:
naming can be improved here. maybe
`const [itemCollapseState, setItemCollapseState] = useState(!name.length);`

Reply:
also, there is another state defined above which is based on the `name` attribute. 
```
const [title, setTitle] = useState(name);
```

I think that's not boolean.